frontend/package.json

```diff
@@ -21,7 +21,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "pretty": "prettier --write ."
```

Comment:
thanks for installing pretier @kharenzze ! :)

Reply:
💪 BTW I also configured the VSCode plugin for prettier, so if you install [it](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) prettier will run on save @petramika

Reply:
>  BTW I also configured the VSCode plugin for prettier, so if you install [it](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) prettier will run on save @petramika

perfect! so then i dont have to manually configure it :) thanks!! :heart_eyes: